### PR TITLE
Full vertical basic functionality + some tests

### PR DESCRIPTION
**Pull request by rocketer13579**

### CODE REVIEW

File: src/main/java/com/revature/byteshare/ingredient/IngredientService.java
Comment (env7295) [inline src/main/java/com/revature/byteshare/ingredient/IngredientService.java:1]: I think the constructor needs to be an autowired here for the repository right?

**Review by aramsing — APPROVED** [review]
Changes look good, what about you Ethan

**Review by env7295 — COMMENTED** [review]
Most of looks really good, I think on Ingredient Service there needs to be an autowired annotation for the repository but that is all.

File: src/main/java/com/revature/byteshare/ingredient/IngredientService.java
Comment (rocketer13579) [reply src/main/java/com/revature/byteshare/ingredient/IngredientService.java:1]: image

it doesn't take any params so it throws an error if its Autowired but I can actually just remove the constructor and instantiate the HttpClient when I declare it

**Pull request closed without merging**